jlab-sensing/soil-power-sensor-firmware
# 80 fix merge errors

**Name/Affiliation/Title**
Steve Taylor

**Purpose of the PR**
Branch was originally supposed to be a hotfix for some clk issues, but has been instead been repurposed to update the calibration procedure and add unit testing for the main file.

**Development Environment**
N/A

**Test Procedure**
Run calibrate_adc, send '0', and see if voltage and current measurements come back.
Run the unit tests and see if it passes.

**Additional Context**
N/A
